### PR TITLE
Demo: ML CI/CD with Google Cloud and Kubeflow

### PR DESCRIPTION
**Pull request by jhammarstedt**
Added README with suggested demo topic

### CODE REVIEW

File: contributions/demo/jhammarstedt-sestys/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Demo:  Machine Learning CI/CD with Github Actions and Kubernetes
+
+
+## Members
+* Johan Hammarstedt, johham@kth.se, Github: jhammarstedt
+
+* Matej Sestak, sestak@kth.se, Github: sestys
```
Comment (cesarsotovalero) [inline contributions/demo/jhammarstedt-sestys/README.md]: Please rename the file to reflect your KTH accounts, ie. change `jhammarstedt-sestys` to `johham-sestak`

File: contributions/demo/jhammarstedt-sestys/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+
+## Members
+* Johan Hammarstedt, johham@kth.se, Github: jhammarstedt
+
+* Matej Sestak, sestak@kth.se, Github: sestys
+
+## Topic
+In this demo we will show how to use Kubernetes and TF serving to develop and deploy a TF/Pytorch model and then automate the full workflow using Github Actions.
+(Alternative idea: if not our original is not accepted we could instead do the same but using Google Cloud)
```
Comment (cesarsotovalero) [inline contributions/demo/jhammarstedt-sestys/README.md]: Google Cloud is more interesting, please change!

**Review by cesarsotovalero — CHANGES_REQUESTED** [review]

File: contributions/demo/jhammarstedt-sestys/README.md (hunk 2 above)
Comment (sestys) [reply contributions/demo/jhammarstedt-sestys/README.md]: Hi @cesarsotovalero, we fixed the folder names and changed the topic a bit to focus specifically on MLops on Google Cloud Platform

**Comment by sestys** [comment]
Hi @cesarsotovalero, is there anything else that's blocking the PR and you want us to change?

**Review by cesarsotovalero — APPROVED** [review]

**Comment by cesarsotovalero** [comment]
> Hi @cesarsotovalero, is there anything else that's blocking the PR and you want us to change?

Your proposal LGTM. 
Looking forward to seeing your demo and learn about this specific MLops tech stack. Good luck!

**Pull request merged**